apps/dashboard/src/components/workflow-editor/steps/in-app/in-app-subject.tsx

```diff
@@ -30,13 +23,13 @@ export const InAppSubject = () => {
         <InputField size="fit">
           <FormItem className="w-full">
             <FormControl>
-              <Editor
+              <FieldEditor
```

Comment:
the font size is too big

![Screenshot 2024-12-31 at 10 05 02](https://github.com/user-attachments/assets/7156d885-a529-41c6-83fe-80d267357dd1)